avocado/core/parser.py

```diff
@@ -24,7 +24,6 @@
 
 from . import exit_codes
 from . import varianter
-from . import settings
 from .future.settings import settings as future_settings
 from .future.settings import ConfigFileNotFound, SettingsError
```

Comment:
I guess with this final step you also mean to rename `future_settings` back into `settings`?

Reply:
That is right, the PR is almost ready for pushing :)

Reply:
@pevogam just FYI, I force pushed here with the @clebergnu's commit that is moving the future.settings to settings.

Reply:
noted, thanks!